app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -0,0 +1,50 @@
+module ServiceOffering
```

Comment:
@lindgrenj6 @syncrou Shouldn't this be under app/services/service_catalog like the other services since service_catalog is the name of the app

Reply:
@mkanoor - We went this direction due to my understanding of a request by @gmcculloug. We can modify the location if that understanding was incorrect.

Reply:
@mkanoor My suggestion was to separate out classes by directory.  I don't see the point of having call the class files packed into a single directory.

Referencing back to [Gourmet Service Objects](http://brewhouse.io/blog/2014/04/30/gourmet-service-objects.html)

>Services go under the app/services directory. I encourage you to use subdirectories for business logic-heavy domains. For instance:
> The file app/services/invite/accept.rb will define Invite::Accept
>while app/services/invite/create.rb will define Invite::Create

Reply:
@mkanoor Do you have any thoughts on response from @gmcculloug?